samples/bluetooth/mesh/light/overlay-dfu.conf

```diff
@@ -12,12 +12,17 @@ CONFIG_MCUMGR_CMD_IMG_MGMT=y
 CONFIG_MCUMGR_SMP_BT=y
 CONFIG_MCUMGR_SMP_BT_AUTHEN=n
 
-CONFIG_BT_L2CAP_TX_MTU=252
-CONFIG_BT_BUF_ACL_RX_SIZE=256
+CONFIG_BT_L2CAP_TX_MTU=190
+CONFIG_BT_BUF_ACL_RX_SIZE=252
+CONFIG_BT_DATA_LEN_UPDATE=y
+CONFIG_BT_AUTO_DATA_LEN_UPDATE=y
 
 # Enable Extended Advertiser to advertise BT SMP service
 CONFIG_BT_EXT_ADV=y
 CONFIG_BT_EXT_ADV_MAX_ADV_SET=4
```

Comment:
The main config file for this sample has RELAY enabled, overlay will merge into that. With EXT ADV defined, I think the value of CONFIG_BT_EXT_ADV_MAX_ADV_SET may be less than needed. Not the topic of this PR but, what about a separate advertiser for GATT?

Reply:
This change should be removed once we globally change adv sets configuration for mesh. But this is not done yet.

Reply:
Sorry, it won't be removed, but will have to be aligned with the global change.

Reply:
Ok. If we don't have a task for it, can you create one?